pkg/fuse/context.go

```diff
@@ -58,7 +62,9 @@ func (fs *fileSystem) newContext(cancel <-chan struct{}, header *fuse.InHeader)
 	ctx.canceled = false
 	ctx.cancel = cancel
 	ctx.header = header
+	ctx.checkPermission = fs.conf.NonDefaultPermission
```

Comment:
When uid is 0, checkPermission should be false

Reply:
done